gui/middleware/notifier.py

```diff
@@ -369,12 +369,12 @@ def __gpt_labeldisk(self, type, devname, swapsize=2):
         else:
             # The GPT header takes about 34KB + alignment, round it to 100
             if size - 100 <= swapgb * 1024 * 1024:
-                raise MiddlewareError('Your disk size must be higher than %dGB' % (swapgb, ))
+                raise MiddlewareError('Disk size must be larger than %dGB' % (swapgb, ))
             # HACK: force the wipe at the end of the disk to always succeed. This
             # is a lame workaround.
             self._system("dd if=/dev/zero of=/dev/%s bs=1m oseek=%s" % (
                 devname,
-                size / 1024 - 32,
+                (size / 1048576) - 32,
```

Comment:
I swear I tested this and found it not erasing the backup GPT.  But I have been wrong before.

Looking at this again, scaling it once to K and then again to M is needlessly confusing.  So scale it once, to meg on line 366:
`size = int(re.sub(r'\s+', ' ', p1.communicate()[0]).split()[2]) / (1024*1024)`
then line 377 becomes much more clear:
`size - 32,`

Line 371 also jumps around in units, and appears to be more complicated than it needs to be.  With this change, it could be
`if size - 1 <= swapgb * 1024:`

For clarity, it might be worth just doing all the scaling into different variables, then using them:

`size = re.sub(r'\s+', ' ', p1.communicate()[0]).split()[2]
sizem = int(size / (1024 * 1024))
sizeg = int(size / (1024 * 1024 * 1024))`

Then use sizem for the dd parameter and sizeg for the check for space.